cooltools/cli/cis_expected.py

```diff
@@ -35,39 +35,21 @@
     type=str,
     required=False,
 )
-@click.option(
-    "--contact-type",
-    "-t",
-    help="compute expected for cis or trans region of a Hi-C map."
-    "trans-expected is calculated for pairwise combinations of specified regions.",
-    type=click.Choice(["cis", "trans"]),
-    default="cis",
-    show_default=True,
-)
 @click.option(
     "--view",
     "--regions",
-    help="Path to a 3 or 4-column BED file containing genomic regions"
-    " for which expected will be calculated. Region names are stored"
-    " optionally in a 4th column, otherwise UCSC notation is generated."
-    " When not specified, expected is calculated for all chromosomes."
-    " Trans-expected is calculated for all pairwise combinations of regions,"
-    " provided regions have to be sorted."
-    " Note that '--regions' is the deprecated name of the option. Use '--view' instead. ",
+    help="Path to a 3 or 4-column BED file with genomic regions"
+    " to calculated cis-expected on. When region names are not provided"
+    " (no 4th column), UCSC-style region names are generated."
+    " Cis-expected is calculated for all chromosomes, when this is not specified."
+    " Note that '--regions' is the deprecated name of the option. Use '--view' instead.",
     type=click.Path(exists=True),
     required=False,
 )
-@click.option(
-    "--balance/--no-balance",
-    help="Apply balancing weights to data before calculating expected."
-    "Bins masked in the balancing weights are ignored from calcualtions.",
```

Comment:
that's how you trigger raw expected caculations now:
```sh
cooltools cis-expected --clr-weight-name ""  CN.mm9.1000kb.cool
```
I don't like it much, but API is streamlined now ! no more `balance` and `clr_weight_name` at the same time